bin/gagarin

```diff
@@ -38,44 +40,51 @@ program
   .option('-T, --startup-timeout <ms>' ,'server startup timeout [5000]', parse10, 5000)
   .option('-U, --build-timeout <ms>' ,'meteor building timeout [120000]', parse10, 120000)
   .option('-O, --velocity <url>', 'report results to velocity at given url')
-  .option('-p, --parallel <number>', 'run test suites in parallel', parse10, 0)
+  .option('-p, --parallel <number>', 'run test suites in parallel', parse10, 0);
 
 program.name = 'gagarin';
 
 program.parse(process.argv);
 
 var gagarin = new Gagarin(program);
-var pathToTests = program.args[0] || path.join(program.pathToApp, 'tests', 'gagarin');
+var pattern = program.args[0];
 
 // set verbose mode if necessary
 logs.setVerbose(program.verbose);
 logs.setSilentBuild(program.muteBuild);
 
-if (!fs.existsSync(pathToTests)) {
-  console.warn('looking for tests in ' + pathToTests + ' but this path does not exist');
-  process.exit(1);
+// fallback to default test dir if there is no pattern provided
+if (!pattern) {
+  var appPath = program.pathToApp.replace(/\\/g, '/');
+  pattern = [appPath, 'tests', 'gagarin', '**/*.{js,coffee}'].join('/');
```

Comment:
@smeijer Shouldn't we use `path.join` here to make sure it also works on windows?